Packs/CommonScripts/Scripts/FileEnrichment/FileEnrichment.yml

```diff
@@ -0,0 +1,350 @@
+args:
+- name: file_hash
+  description: 'Hash of the file. Supported types are: MD5, SHA1, SHA256, and SHA512.'
+- name: enrichment_brands
+  description: |-
+    Integrations brands to use for running the file reputation command. If not provided, the command will run for all available integrations that contain the file reputation command.
+    For multi-select, provide a comma-separated list. For example: "VirusTotal (API v3),WildFire-v2".
+  auto: PREDEFINED
+  predefined:
+  - 'VirusTotal (API v3)'
+  - 'WildFire-v2'
+  - 'Cortex Core - IR'
+- name: verbose
+  description: Whether to retrieve human readable entry for every command or only the final result. Set to true to get a human-readable entry for every command. Set to false to get a human-readable summary of the final result.
+  auto: PREDEFINED
+  predefined:
+  - 'true'
+  - 'false'
+- name: external_enrichment
+  description: Whether to run additional external indicator enrichment commands. Set to true to enrich with information from the specified source brands. Set to false to only query for existing indicators in the Threat Intelligence Module (TIM).
+  auto: PREDEFINED
+  predefined:
+  - 'true'
+  - 'false'
+comment: This script gathers file reputation data from multiple integrations and returns a File entity with consolidated information to the context output.
```

Comment:
This script gathers file reputation data from multiple integrations and returns a file entity with consolidated information to the context output.